xla/pjrt/c/pjrt_c_api.h

```diff
@@ -79,7 +79,7 @@ PJRT_DEFINE_STRUCT_TRAITS(PJRT_Extension_Base, next);
 // Changes include:
 // * Adding a new field to the PJRT_Api or argument structs
 // * Renaming a method or argument (doesn't affect ABI)
-#define PJRT_API_MINOR 57
+#define PJRT_API_MINOR 58
```

Comment:
Please add to the `xla/pjrt/c/CHANGELOG.md`

Reply:
Addressed in https://github.com/openxla/xla/pull/20429/commits/5017a0ed720492ecf8d47cafd8d2b09d7222cbf8